_includes/footer_custom.html

```diff
@@ -0,0 +1,10 @@
+<p>
+  {%- if site.email -%}
+  Questions? Be in touch with our team at <a class="u-email" href="mailto:{{ site.email }}">{{ site.email }}</a>
+  {%- endif -%}
+</p>
+<p>
+  {%- if site.homepage -%}
+  Learn more about YourBase at our <a href="{{ site.homepage }}">homepage</a>.
```

Comment:
Same here (but referrer is okay because we trust our own domain and want the analytics):

```suggestion
  Learn more about YourBase at our
  <a href="{{ site.homepage }}" target="_blank" rel="noopener">homepage</a>.
```